nav2_bt_navigator/behavior_trees/navigate_w_replanning_and_recovery.xml

```diff
@@ -16,7 +16,7 @@
           </Fallback>
         </RateController>
         <RecoveryNode number_of_retries="1" name="FollowPath">
-          <FollowPath path="{path}"/>
+          <FollowPath path="{path}" controller_name="FollowPath"/>
```

Comment:
Can we extrapolate here [what's decided for the planner](https://github.com/ros-planning/navigation2/pull/1269/files#r340749134)? 

We'd need to replace `controller_name` to something else. One option is `control_property` if we're thinking the input describes some *quality* about how the path should be followed.

Reply:
Yes, of course. I’ll reflect whatever we decide there, here as well

Reply:
One consequence of this I really dont like is that now a line reads: `controller_properties[action->controller_property]->doThing()`. Its **very** unclear that `controller_properties_` means a vector of properties and not a set of properties about 1 planner. 

Lets merge this now but we need to discuss this. I'd like to change the variable name back to `planner_names_` inside the server so that this is abundantly clear, but still keep `property` everywhere else.

Is that something you'd agree to?

Also see below, I changed everything. Still need the second review on this one to merge

Reply:
https://github.com/ros-planning/navigation2/issues/1330

Ticket to continue dialog